content/pages/security.md

```diff
@@ -4,9 +4,21 @@ save_as: security.html
 template: security
 
 ## How to report a security issue
-If you believe you have discovered a vulnerability in Solr, you may first want to consult the [list of known false positives](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) to make sure you are reporting a real vulnerability.
-Then please disclose responsibly by following [these ASF guidelines](https://www.apache.org/security/) for reporting.
 
+### CVEs in Solr dependencies
+
+Solr depends on lots of other open-source software -- "dependencies".  If a CVE is published (a publicly identified vulnerability) against one of them, the Solr project will review it to see if it's actually exploitable in Solr -- usually they aren't.  Please review the [officially published non-exploitable vulnerabilities](https://cwiki.apache.org/confluence/display/SOLR/SolrSecurity#SolrSecurity-SolrandVulnerabilityScanningTools) before taking any steps.  If you **don't** see a CVE there, you should take the following steps:
+1. Search through the [Solr users mailing list](https://lists.apache.org/list.html?users@solr.apache.org) to see if anyone else has brought up this dependency CVE.
+1. If no one has, then please do [subscribe to the users mailing list](https://solr.apache.org/community.html#mailing-lists-chat) and then send an email asking about the CVE.
+
+### Exploits found in Solr
+
+The Solr PMC will not accept the output of a vulnerability scan as a security report.
```

Comment:
This line is only applicable to "CVEs in Solr dependencies".  You could begin that section with this sentence.